### PR TITLE
Added info on breaking changes to release notes for 2.0.0

### PR DESCRIPTION
**Pull request by dmccoystephenson**
## Problem
The switch to J2735 2020 means that 2016 TIMs will no longer be processed successfully by the ACM. There is no mention of this in the release notes.

## Solution
Information on breaking changes has been added to the release notes.

### CODE REVIEW

File: docs/Release_notes.md [hunk 1]
```diff
@@ -19,6 +19,9 @@ Known Issues:
 - The do_kafka_test.sh script in the project's root directory is currently not running successfully. The issue is being investigated and will be addressed in a future update.
 - According to Valgrind, a minor memory leak has been detected. The development team is aware of this and is actively working on resolving it.
 
+Breaking Changes
+- Users should note that due to the switch to J2735 2020, TIMs conforming to J2735 2016 will no longer be encoded/decoded correctly. This is due to the fact that the J2735 2020 standard has renamed some fields for TIMs. Users should ensure that their TIMs are updated to conform to the J2735 2020 standard.
```
Comment (payneBrandon) [inline docs/Release_notes.md]: Let's add to this last sentence saying that the ODE updates handle this change already, but any outside calling applications will need to update their systems.

**Review by payneBrandon — CHANGES_REQUESTED** [review]
One comment, looks good!

File: docs/Release_notes.md (hunk 1 above)
Comment (dmccoystephenson) [reply docs/Release_notes.md]: The last sentence has been rephrased & added to for clarity.

**Comment by dmccoystephenson** [comment]
This is ready for re-review. @payneBrandon

**Review by drewjj — APPROVED** [review]
Looks good

**Review by drewjj — APPROVED** [review]

**Review by dan-du-car — APPROVED** [review]

**Review by payneBrandon — APPROVED** [review]
lgtm!

**Review by dan-du-car — APPROVED** [review]

**Pull request merged**